README.md

```diff
@@ -44,12 +44,62 @@ use({
       ...,
       adapters = {
         require("neotest-vitest")
-        }
+      }
     })
   end
 })
 ```
 
+## Configuration
+```lua
+{
+  "nvim-neotest/neotest",
+  dependencies = {
+    ...,
+    "marilari88/neotest-vitest",
+  },
+  config = function()
+    require("neotest").setup({
+      ...,
+      adapters = {
+        require("neotest-vitest") {
+          -- Filter directories when searching for test files. Useful in large projects (see Filter directories notes).
+          filter_dir = function(name, rel_path, root)
```

Comment:
We should add also the other config options. But I can do myself later. You can ignore this comment

Reply:
Hmm, I intended to open another PR to add some more info in README. Let me know.

Reply:
It would be great! 😃